### PR TITLE
Fix nightly msi file name

### PR DESCRIPTION
**Pull request by t3chguy**

### CODE REVIEW

File: src/desktop_develop.ts [hunk 1]
```diff
@@ -424,7 +424,7 @@ export default class DesktopDevelopBuilder {
             for (const f of await getMatchingFilesInDir(path.join(repoDir, 'dist'), /\.msi$/)) {
                 await copyAndLog(
                     path.join(repoDir, 'dist', f),
-                    path.join(this.appPubDir, 'install', 'win32', archDir, 'msi', f),
+                    path.join(this.appPubDir, 'install', 'win32', archDir, 'msi', 'Element Nightly Setup.msi'),
```
Comment (duxovni) [inline src/desktop_develop.ts:427]: I'm a bit confused; is there only one `.msi` file in the directory we're copying from, or are we repeatedly overwriting `Element Nightly Setup.msi`?

Comment (t3chguy) [reply src/desktop_develop.ts:427]: Yeah, this is just the pattern we use for that, same as above for the `.exe`, the callback will only really fire once

Comment (duxovni) [reply src/desktop_develop.ts:427]: I see. Might be nice at some point to get rid of the `for` loop, and replace `getMatchingFilesInDir` with a function that returns the one matching file and throws an error if there isn't exactly one match. Up to you whether that's worth holding up this PR for, though; I'll go ahead and approve.

**Review by duxovni — APPROVED** [review]

File: src/desktop_develop.ts (hunk 1 above)
Comment (t3chguy) [reply src/desktop_develop.ts:427]: I have lots of plans to consolidate and tidy the code here :)

**Pull request merged**